lib/string-checker.js

```diff
@@ -130,7 +130,12 @@ StringChecker.prototype = {
 
         this._configuredRules.forEach(function(rule) {
             errors.setCurrentRule(rule.getOptionName());
-            rule.check(file, errors);
+
+            try {
```

Comment:
> execution time like 5% lower in branch than in master

Hm, i interpret this phrase as this changes would add 5%. So you somehow make it faster? :anguished: